WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -23,7 +24,10 @@ @implementation XCUIElement (FBIsVisible)
 
 - (BOOL)fb_isVisible
 {
-  return self.fb_lastSnapshot.fb_isVisible;
+  if (SYSTEM_VERSION_LESS_THAN(@"11.0") || [FBConfiguration shouldUseTestManagerForVisibilityDetection]) {
```

Comment:
Unlikely... at least on iOS9 snapshot's hitPoint property and hittable property of XCElement return different resulsts